velox/docs/functions/spark/datetime.rst

```diff
@@ -163,6 +163,14 @@ These functions support TIMESTAMP and DATE input types.
 
     Alias for ``unix_timestamp(string, format) -> integer``.
 
+.. spark:function:: unix_date(date) -> integer
+
+    Returns the number of days since 1970-01-01.
```

Comment:
Would you generate the docs and check formatting? It looks like you might be missing :: before the examples.

Reply:
Done, thanks!